test/Concurrency/predates_concurrency.swift

```diff
@@ -86,7 +90,7 @@ func testCallsWithAsync() async {
   onMainActorAlways() // expected-error{{expression is 'async' but is not marked with 'await'}}
   // expected-note@-1{{calls to global function 'onMainActorAlways()' from outside of its actor context are implicitly asynchronous}}
 
-  let _: () -> Void = onMainActorAlways // expected-error{{converting function value of type '@MainActor () -> ()' to '() -> Void' loses global actor 'MainActor'}}
+  let _: () -> Void = onMainActorAlways // expected-warning{{converting function value of type '@MainActor () -> ()' to '() -> Void' loses global actor 'MainActor'}}
```

Comment:
I wonder if we should also allow the assignment of a non-Sendable closure to a property whose type requires that it be sendable, if the property is a member of a pre-concurrency type / module?

Reply:
This already seems to work on `main` for Swift 5 without preconcurrency:

```swift
struct hello {
  var clos: (@Sendable () -> Void)? = nil
}

func asdf(f: @escaping () -> Void) -> hello {
  var h = hello()
  h.clos = f // warning: assigning non-sendable parameter 'f' to a @Sendable closure
}
```